README.md

```diff
@@ -70,16 +70,20 @@ Install in editable mode and with extra developer dependencies into your virtual
 
 Configure the `pre-commit` hooks:
 
-    pre-commit install && pre-commit install -t pre-push
+    pre-commit install
+    pre-commit install -t pre-push
+    pre-commit install -t prepare-commit-msg
+    pre-commit install -t commit-msg
 
+These hooks check code formatting and also aborts commits that contain secrets such as passwords or API keys. However, a one time setup is required in your global Git configuration. See [the wiki entry on Git Secrets](https://github.com/NASA-PDS/nasa-pds.github.io/wiki/Git-and-Github-Guide#git-secrets) to learn how.
 
 ### Packaging
 
 To isolate and be able to re-produce the environment for this package, you should use a [Python Virtual Environment](https://docs.python.org/3/tutorial/venv.html). To do so, run:
 
     python -m venv venv
 
-Then exclusively use `venv/bin/python`, `venv/bin/pip`, etc. (It is no longer recommended to use `venv/bin/activate`.)
```

Comment:
@nutjob4life I am curious about this sentence "(It is no longer recommended to use `venv/bin/activate`.)" ?
I am still using that frequently, , actually `source ./venv/bin/activate`, although pycharm does it for us

Reply:
I had read somewhere that as part of Python's "explicit > implicit" philosophy, using explicit paths (`venv/bin/python`) was superior to doing `bin/activate` and then relying on shell paths to find `python`.

However, I deleted that sentence because others recommended the _opposite_, especially if your Python code does `system` or `Popen` on `python` (without using `sys.executable`) and it finds the wrong Python.

So I deleted that sentence.

Reply:
Seems like a stretch to extend that philosophy to non-code CLI commands (vs running python from a shell from python) anyway

Reply:
@tloubrieu-jpl are we OK with merging this?